src/containers/shared/components/Amount.tsx

```diff
@@ -44,4 +48,35 @@ export const Amount = ({
       />
     </span>
   )
+
+  const mptID = isMPT ? (value as ExplorerAmount).currency : null
+
+  // fetch MPTIssuance only if isMPT is true
+  const { data: mptIssuanceData } =
+    useQuery<MPTIssuanceFormattedInfo>(
+      ['getMPTIssuanceScale', mptID],
+      async () => {
+        const info = await getMPTIssuance(rippledSocket, mptID)
+        return formatMPTIssuanceInfo(info)
+      },
+      {
+        onError: (e: any) => {
+          trackException(`mptIssuance ${mptID} --- ${JSON.stringify(e)}`)
+        },
+        enabled: isMPT,
+      },
+    ) || {}
+
+  // if amount is MPT type, we need to fetch the scale from the MPTokenIssuance
+  // object so we can show the scaled amount
+  if (isMPT && typeof value !== 'string') {
+    if (mptIssuanceData) {
+      const scale = mptIssuanceData.assetScale ?? 0
+      const scaledAmount = parseInt(amount as string, 10) / 10 ** scale
```

Comment:
I did create a function to convert to scaled price (tested to work with big numbers):
https://github.com/ripple/explorer/blob/staging/src/containers/shared/components/Transaction/OracleSet/parser.ts#L7-L16

Wonder if it can be reused here

Reply:
fixed